zcash_client_backend/CHANGELOG.md

```diff
@@ -191,10 +194,19 @@ and this library adheres to Rust's notion of
 - `zcash_client_backend::keys`:
   - `DerivationError::Orchard` is now only available when the `orchard` feature
     is enabled.
+  - `UnifiedSpendingKey::address` now takes an argument that specifies the
+    receivers to be generated in the resulting address. Also, it now returns
+    `Result<UnifiedAddress, AddressGenerationError>` instead of
+    `Option<UnifiedAddress>` so that we may better report to the user how
+    address generation has failed.
   - `UnifiedSpendingKey::orchard` is now only available when the `orchard`
     feature is enabled.
+  - `UnifiedSpendingKey::transparent` is now only available when the
+    `transparent-inputs` feature is enabled.
   - `UnifiedFullViewingKey::new` no longer takes an Orchard full viewing key
     argument unless the `orchard` feature is enabled.
+  - `UnifiedFullViewingKey` and `UnifiedSpendingKey` now only contain transparent
+    key components in the case that the `transparent-inputs` feature is enabled.
```

Comment:
This is not new; it's been the case since `UnifiedSpendingKey` was introduced in https://github.com/zcash/librustzcash/commit/dec395a5b0c3bd5ee1fb2661ded06005ce00b81f. Did you mean something else here?

Reply:
It's also ambiguously written. I think it's intended to mean "When the `transparent-inputs` feature is disabled, `UnifiedFullViewingKey` and `UnifiedSpendingKey` omit transparent key components.", but can also be read as "When the `transparent-inputs` feature is enabled, `UnifiedFullViewingKey` and `UnifiedSpendingKey` now contain only transparent key components (and no shielded components)."